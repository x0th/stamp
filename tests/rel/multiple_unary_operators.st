Object f = fn() { return ---1; }
Object.f()
