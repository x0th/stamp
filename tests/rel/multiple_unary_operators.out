STDOUT:
-1
STDERR:
